app/v1/models/request/NinoApplication.scala

```diff
@@ -84,7 +84,7 @@ object NinoApplication {
       officeNumberPath.read[String].filter(commonError("office number"))(validateAgainstRegex(_, officeNumberRegex)) and
       contactNumberPath.readNullable[String].filter(commonError("contact number"))(_.fold(true)(validateAgainstRegex(_, contactNumberRegex))) and
       countryPath.read[Int] and
-      namePath.read[NameModel] and
+      namePath.read[NameModel].map(Seq(_)) and
```

Comment:
Can you update namePath to be `names` please

Reply:
Done